base/src/PipeLine.cpp

```diff
@@ -1,37 +1,87 @@
 #include "stdafx.h"
 #include "PipeLine.h"
 #include "Module.h"
+#include <string>
+#ifndef _WIN64
+#include <pthread.h>
+#endif
+
+#ifdef _WIN64
```

Comment:
the function "setModuleThreadName" and all it's beggage should be moved to Utils.h,cpp and called from here.